src/gui_common/CustomPopupMenu.cs

```diff
@@ -48,7 +43,9 @@ protected virtual void ResolveNodeReferences()
 
     protected override void OnOpen()
     {
-        CreateTween().TweenProperty(this, "rect_size", cachedMinSize, 0.2f)
+        // RectSize = CalculateSize();
```

Comment:
I guess this comment is fine to leave here as a sign that this used to work differently...

Reply:
O shoot, I forgot that was in there. Wasn't intended to be left in, sorry.

Reply:
If it is not too much of a bother, could you make a PR removing that line? It'll be very easy to approve and merge.